multiapps-controller-process/src/main/java/org/cloudfoundry/multiapps/controller/process/steps/DetermineServiceCreateUpdateServiceActionsStep.java

```diff
@@ -32,6 +32,7 @@
 import org.springframework.beans.factory.config.BeanDefinition;
 import org.springframework.context.annotation.Scope;
 import org.springframework.http.HttpStatus;
+import static org.cloudfoundry.multiapps.controller.process.steps.StepsUtil.canDeleteServiceKeys;
```

Comment:
Please don't :smile: 
Just use a normal import and do `return StepsUtil.canDeleteServiceKeys(context)`